borderless/explainer.md

```diff
@@ -0,0 +1,512 @@
+`display_override` and display values on the `manifest.json`. In case no other
+display mode is provided, it will
+[default to standalone mode](https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/web_applications/web_app_proto_utils.cc;drc=4a8573cb240df29b0e4d9820303538fb28e31d84;l=161)
+similarly to WCO and other `display_override` modes.
+
+### 5. Isolated Web App check
+
+Borderless mode is only available for
+[Isolated Web Apps (IWAs)](https://github.com/WICG/isolated-web-apps).
+
+## Related features
+
+- [app-details feature](crbug.com/1225871) (Googlers-only)
+- [go/cros-privacy-indicators-design](go/cros-privacy-indicators-design)
+  (Googlers-only)
+- [additional windowing controls](https://github.com/ivansandrk/additional-windowing-controls/blob/main/awc-explainer.md)
+
+### Displaying app’s origin in the app settings UI
+
+Since without the title bar and the 3-dot menu, the app’s origin won’t be visible
+anymore, it needs to be shown somewhere, in this case in App Settings. This
+work has been completed as part of the [app-details feature](crbug.com/1225871),
+but it is pending its launch.
+
+Displaying the origin is mainly necessary for debugging purposes. Until the app
+details get launched, an alternative way to get the origin of an IWA in
+borderless mode is via the developer console with
+[`document.location.href`](https://developer.mozilla.org/en-US/docs/Web/API/Location/href).
+
+In the future when app details have been launched and origin is available, the
+app settings can be accessed via opening the system’s settings and then looking
+for the right app or by right clicking the app’s icon on the shelf. Or by
+visiting the app's OS-settings in the following type of URL:
+<span style="text-decoration:underline;">chrome://os-settings/app-management/detail?id=&lt;APP_ID></span>
+
+Mock of app settings UI with app’s origin:  
+![App settings UI mock](./images/app-settings-ui.png)
+
+### Privacy indicators
+
+Removing the title bar also removes the area where critical privacy indicators
+would live, e.g. for camera and microphone access. However there is already
+another on-going project to move those privacy indicators to the shelf.
+
+![Example privacy indicator](./images/privacy-indicators-in-shelf.png)
+
+See (Googlers only)
+[go/cros-privacy-indicators-design](go/cros-privacy-indicators-design) for more.
+
+## Other notable details
+
+### “Settings and more” three-dot button
+
+Since the three-dot button (which gives users access to extensions, security
+information about the page, access to cookies, etc.) will be disappearing, the
+most critical information (app’s origin) will be displayed in the app settings.
+
+The menu behind the three-dots button contains the following:
+
+<img src="./images/three-dot-menu.png" alt="Three-dot menu" width="450">
+
+<table>
+  <tr>
+   <td><strong>Button</strong></td>
+   <td><strong>Status</strong></td>
+  </tr>
+  <tr>
+   <td>App info</td>
```

Comment:
I think this table is going into too much detail about specifics for one OS, and can be moved into a separate design doc, rather than the W3C explainer.

You could just generally say that user agents will need to drop or find a home for these sort of items that would normally appear on the app menu.